dotnet-authserver/src/TeacherIdentity.AuthServer/Pages/Admin/Users.cshtml

```diff
@@ -124,12 +124,12 @@
                     <govuk-pagination>
                         @if (Model.PreviousPage.HasValue)
                         {
-                            <govuk-pagination-previous asp-page="Users" asp-all-route-data=@Model.GetPage((int)Model.PreviousPage!)/>
+                            <govuk-pagination-previous asp-page="Users" asp-all-route-data=@Model.GetPage((int)Model.PreviousPage!) asp-route-usersearch="@Model.UserSearch" />
```

Comment:
Can we add `UserSearch` to the `GetPage` method that way we've got everything in one place?